sdk/go/aws/rds/getEngineVersion.go

```diff
@@ -54,7 +62,9 @@ func GetEngineVersion(ctx *pulumi.Context, args *GetEngineVersionArgs, opts ...p
 // A collection of arguments for invoking getEngineVersion.
 type GetEngineVersionArgs struct {
 	// DB engine. Engine values include `aurora`, `aurora-mysql`, `aurora-postgresql`, `docdb`, `mariadb`, `mysql`, `neptune`, `oracle-ee`, `oracle-se`, `oracle-se1`, `oracle-se2`, `postgres`, `sqlserver-ee`, `sqlserver-ex`, `sqlserver-se`, and `sqlserver-web`.
-	Engine string `pulumi:"engine"`
+	// * * `filter` - (Optional) One or more name/value pairs to filter off of. There are several valid keys, for a full reference, check out [describe-db-engine-versions in the AWS CLI reference][1].
```

Comment:
I don't think the CLI reference link is valid, and it looks like the comment was added to the wrong option (engine).

Reply:
this is a bug in tfgen then - nothing gets added to our Go SDK directly - it comes from the TF Provider docs